rainbows.rb

```diff
@@ -27,3 +27,32 @@ def mix_colors(color1,color2)
 
 new_color = mix_colors(red,green)
 puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+# to-do 1
+rainbow = [
+ {:red => {:r => 255, :g => 0, :b => 0}},
+ {:orange => {:r => 255, :g => 127, :b=>0}},
+ {:yellow => {:r => 255, :g => 255, :b=>0}},
+ {:green => {:r => 0, :g => 255, :b=>0}},
+ {:blue => {:r => 0, :g => 0, :b=>255}},
+ {:purple => {:r => 75, :g => 0, :b=>130}},
+ {:violet => {:r => 143, :g => 0, :b=>255}}]
+
```

Comment:
Also, I _think_ that there's a standard way we should define hashes when using symbols that you aren't doing here. This works, but I think isn't standard convention.

I believe it should be 

```
{red: {r: 255, g:0, b:0}},
```